src/ui.rs

```diff
@@ -358,6 +358,11 @@ impl<'a> CsvTable<'a> {
                 content += format!(" [Echo {column_name} ↵]").as_str();
             }
 
+            // Ignore case option
+            if state.ignore_case {
+                content += format!(" (ignore-case: {})", state.ignore_case).as_str();
```

Comment:
Is the value of `ignore_case` always true when printed? If so, I think it can make sense to simplify to just show the `ignore-case` without the value, more like a label . I slightly prefer square bracket for consistency with the rest of the status message.

Reply:
Also formatted it per your suggestion.